tests/CMakeLists.txt

```diff
@@ -0,0 +1,72 @@
+enable_testing()
```

Comment:
It seems to me, reading around this a bit, that `enable_testing` includes `CTest`, which again has this `BUILD_TESTING` option.

Are you aware of this? If so, why not use it instead of `BUILD_TESTS`?

Reply:
I wasn't. I copy/pasted the logic out of the main.cmake file.

Reply:
Interesting (not against it), could you explain a bit further what issues CTest has with `ccache` ?